basis/html/templates/chloe/compiler/compiler.factor

```diff
@@ -74,17 +79,38 @@ SYMBOL: string-context?
 
 ERROR: tag-not-allowed-here ;
 
+CONSTANT: self-closing-tags {
```

Comment:
Seemed like a bad complexity trade-off to go that route; the list is only consulted at template compile time, which is relatively rare, so I wasn't convinced that the time and memory to generate the hash set was going to be made up by savings on compile time later. That said, I don't feel strongly about this. Just let me know.